packages/dev/core/src/Cameras/Inputs/arcRotateCameraPointersInput.ts

```diff
@@ -231,24 +230,23 @@ export class ArcRotateCameraPointersInput extends BaseCameraPointersInput {
      * press.
      * @param evt
      */
-    protected onButtonDown(evt: IPointerEvent): void {
+    public onButtonDown(evt: IPointerEvent): void {
         this._isPanClick = evt.button === this.camera._panningMouseButton;
     }
 
     /**
      * Called each time a new POINTERUP event occurs. Ie, for each button
      * release.
-     * @param evt
      */
-    protected onButtonUp(evt: IPointerEvent): void {
+    public onButtonUp(): void {

```

Comment:
This is supposed to override what's in BaseCameraPointersInput so I feel like this should match what is in that class, either by re-adding `evt` or removing it from the BaseCameraPointersInput class version.